<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Pankaj" failures="2" tests="4" name="testpackBewkoof.ToVerifyShirtSection" time="40.573" errors="0" timestamp="2023-04-10T21:05:41 IST" skipped="0">
  <testcase classname="testpackBewkoof.ToVerifyShirtSection" name="verifyTitleofShirtSectionPage" time="8.344"/>
  <system-out/>
  <testcase classname="testpackBewkoof.ToVerifyShirtSection" name="veriftShirtDetailsAfterClicking" time="10.093">
    <failure message="expected [https://www.bewakoof.com/p/mens-red-groovin-graphic-printed-oversized-t-shirt] but found [https://www.bewakoof.com/p/men-gulf-coast-black-printed-t-shirt]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [https://www.bewakoof.com/p/mens-red-groovin-graphic-printed-oversized-t-shirt] but found [https://www.bewakoof.com/p/men-gulf-coast-black-printed-t-shirt]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at testpackBewkoof.ToVerifyShirtSection.veriftShirtDetailsAfterClicking(ToVerifyShirtSection.java:73)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- veriftShirtDetailsAfterClicking -->
  <system-out/>
  <testcase classname="testpackBewkoof.ToVerifyShirtSection" name="verifyTitleofShirtSectionPage" time="10.124"/>
  <system-out/>
  <testcase classname="testpackBewkoof.ToVerifyShirtSection" name="veriftShirtDetailsAfterClicking" time="12.012">
    <failure message="expected [https://www.bewakoof.com/p/mens-red-groovin-graphic-printed-oversized-t-shirt] but found [https://www.bewakoof.com/p/men-gulf-coast-black-printed-t-shirt]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [https://www.bewakoof.com/p/mens-red-groovin-graphic-printed-oversized-t-shirt] but found [https://www.bewakoof.com/p/men-gulf-coast-black-printed-t-shirt]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at testpackBewkoof.ToVerifyShirtSection.veriftShirtDetailsAfterClicking(ToVerifyShirtSection.java:73)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- veriftShirtDetailsAfterClicking -->
  <system-out/>
</testsuite> <!-- testpackBewkoof.ToVerifyShirtSection -->
